pennylane/ops/qubit/matrix_ops.py

```diff
@@ -136,6 +136,8 @@ def __init__(
         U_shape = qml.math.shape(U)
         dim = 2 ** len(wires)
 
+        self._issparse = False
```

Comment:
Do we have plans to always have the attribute issparse? In case of yes, can we have it in the base class (Operation) defaulting to false?

Reply:
`_issparse` is a private implementation detail of `QubitUnitary`.  The public interface in `Operator.has_matrix` and `Operator.has_sparse_matrix`.